_typos.toml

```diff
@@ -0,0 +1,8 @@
+[default]
+extend-ignore-re = ["(?Rm)^.*(#|//)\\s*spellchecker:disable-line$"]
+
+[default.extend-identifiers]
+iNdEx = "iNdEx"
+
+[files]
+extend-exclude = ["**/go.mod", "**/go.sum", "**/*.pb.go"]
```

Comment:
I added some excludes here to avoid checking package hashes and generated files. There was already one false-positive in `go.sum`.